html/laravel/app/Models/Task.php

```diff
@@ -128,4 +128,9 @@ public function task_resolution()
     {
         return $this->belongsTo(TaskResolution::class, 'task_resolution_id');
     }
+    
+    public function task_comments()
+    {
+        return $this->hasMany(Comment::class, 'id', 'task_id');
```

Comment:
おそらくこれの間違いだと思うので直しましょう。
```suggestion
        return $this->hasMany(Comment::class, 'task_id');
```